app/logic/Mail/IMAP/IMAPAccount.ts

```diff
@@ -234,10 +247,50 @@ export class IMAPAccount extends MailAccount {
 
   async hasCapability(capa: string): Promise<boolean> {
     let conn = await this.connection();
-    let capabilities = await conn.capabilities;
+    // conn.capabilities doesn't work; it's an object property,
+    // and JPC doesn't notice direct changes to properties.
+    let capabilities = await conn.run('CAPABILITY');
```

Comment:
We shouldn't re-run the network command every time some function needs the result. Can you fix that the function works and uses the result stored on the backend? Obviously, it should be the result from the last run of `CAPABILITY`, e.g. be different after login.